scripts/hudson/quickstart.sh

```diff
@@ -156,7 +180,61 @@ function build_apache-karaf {
     exit -1
   fi
 }
+function clone_as {
+  REPO="https://github.com/wildfly/wildfly.git"
+  echo "Cloning AS sources from $REPO"
 
+  cd $WORKSPACE
+
+  if [ -d jboss-as ]; then
+    rm -rf jboss-as # start afresh
+  fi
+
+  git clone $REPO jboss-as || fatal "git clone $REPO failed"
+
+  if [ $REDUCE_SPACE = 1 ]; then
+    echo "Deleting git dir to reduce disk usage"
+    rm -rf .git
+  fi
+
+  cd $WORKSPACE
+}
+function build_as {
+  AS_BRANCH=27.0.0.Alpha1
+  echo "Building WildFly $AS_BRANCH"
+
+  cd $WORKSPACE/jboss-as
+
+  git checkout $AS_BRANCH || fatal "git checkout $AS_BRANCH failed"
+
+  # building WildFly
+  [ "$_jdk" -lt 17 ] && export MAVEN_OPTS="-XX:MaxMetaspaceSize=512m -XX:+UseConcMarkSweepGC $MAVEN_OPTS"
+  [ "$_jdk" -ge 17 ] && export MAVEN_OPTS="-XX:MaxMetaspaceSize=512m $MAVEN_OPTS"
+  JAVA_OPTS="-Xms1303m -Xmx1303m -XX:MaxMetaspaceSize=512m $JAVA_OPTS" ./build.sh clean install -B -DskipTests -Dts.smoke=false $IPV6_OPTS -Dversion.org.jboss.narayana=${NARAYANA_CURRENT_VERSION} "$@"
+  [ $? -eq 0 ] || fatal "AS build failed"
+
+  WILDFLY_VERSION_FROM_JBOSS_AS=`awk '/wildfly-parent/ { while(!/<version>/) {getline;} print; }' ${WORKSPACE}/jboss-as/pom.xml | cut -d \< -f 2|cut -d \> -f 2`
+  echo "AS version is ${WILDFLY_VERSION_FROM_JBOSS_AS}"
+  JBOSS_HOME=${WORKSPACE}/jboss-as/build/target/wildfly-${WILDFLY_VERSION_FROM_JBOSS_AS}
+  export JBOSS_HOME=`echo  $JBOSS_HOME`
+
+  # init files under JBOSS_HOME before AS TESTS is started
+  init_jboss_home
+
+  cd $WORKSPACE
+}
+function init_jboss_home {
+  [ -d $JBOSS_HOME ] || fatal "missing AS - $JBOSS_HOME is not a directory"
+  echo "JBOSS_HOME=$JBOSS_HOME"
+  cp ${JBOSS_HOME}/docs/examples/configs/standalone-xts.xml ${JBOSS_HOME}/standalone/configuration
+  cp ${JBOSS_HOME}/docs/examples/configs/standalone-rts.xml ${JBOSS_HOME}/standalone/configuration
+  # configuring bigger connection timeout for jboss cli (WFLY-13385)
+  CONF="${JBOSS_HOME}/bin/jboss-cli.xml"
+  sed -e 's#^\(.*</jboss-cli>\)#<connection-timeout>30000</connection-timeout>\n\1#' "$CONF" > "$CONF.tmp" && mv "$CONF.tmp" "$CONF"
+  grep 'connection-timeout' "${CONF}"
+  #Enable remote debugger
+  echo JAVA_OPTS='"$JAVA_OPTS -agentlib:jdwp=transport=dt_socket,address=8787,server=y,suspend=n"' >> "$JBOSS_HOME"/bin/standalone.conf
```

Comment:
@marcosgopen FYI I think this is the reason the `xts-ssl` quickstart is failing at the point it is in your CI build number 111. There tries to be two servers running with port 8787

Reply:
I had the same idea and I have commented those line, but no luck. It is about the wildfly quickstart that are downloaded here (https://github.com/jbosstm/quickstart/blob/master/XTS/ssl/xts-over-ssh.sh) I think and I am working on it.

Reply:
I think removing the exporting of the port 8787 could still be right. Could it be some other problem?

In more abstract terms, are two servers expected to be running at the same time? If so then having two servers configured with -agentlib:jdwp=transport=dt_socket,address=8787,server=y,suspend=n" at the same time will cause a problem. I can't remember explicitly what I saw but it was something like a bind exception. I can try to get the log message if it's helpful?

Reply:
Yes, sorry you are right. I remember the bind exception. Thank you, I will change the debug port then.